ONSdigital/response-operations-ui
# Ras 303 change sample summary call

# What and why?
There was a redundant call to get the sample summaries for a group of collection exercises that needed removal.

# How to test?
Deploy and run acceptance tests. Also, make sure you can access the surveys page, click on colllection exercises, and view their info

# Jira
[Card](https://jira.ons.gov.uk/browse/RAS-303)
